src/style.css

```diff
@@ -1,3 +1,84 @@
+:root {
+  --text-color-second: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav,
+.nav__list,
+.nav__item {
+  display: flex;
+}
+
+.nav {
+  padding: 0;
+  margin: 0 50px;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.header-logo {
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__link {
+  margin: 0 10px;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+  color: black;
+}
+
+.nav__link.last {
+  margin-right: 0;
+}
+
+.nav__link.first {
+  margin-left: 0;
+}
+
+.nav__link.is-active {
+  position: relative;
+  color: var(--text-color-second);
+}
+
+.nav__link:hover {
+  color: var(--text-color-second);
+}
+
+.nav__link.is-active::after {
+  content: "";
+  background-color: var(--text-color-second);
+  position: absolute;
```

Comment:
Pay attention to the CSS properties order
https://9elements.com/css-rule-order/